synapse/handlers/devicemessage.py

```diff
@@ -100,14 +108,21 @@ def send_device_message(self, sender_user_id, message_type, messages):
 
         message_id = random_string(16)
 
+        context = get_active_span_text_map()
+
         remote_edu_contents = {}
         for destination, messages in remote_messages.items():
-            remote_edu_contents[destination] = {
-                "messages": messages,
-                "sender": sender_user_id,
-                "type": message_type,
-                "message_id": message_id,
-            }
+            with start_active_span("to_device_for_user"):
```

Comment:
we're not really doing much inside this span. Is it worthwhile?

Reply:
It's useful for connecting the device message to a particular point in time. This could be done with a log but this is easier to visualize:

![Screenshot from 2019-08-20 11-26-54](https://user-images.githubusercontent.com/23084468/63339934-dd04a000-c33d-11e9-8a37-6125d5244e39.png)

(It's missing the receiving side of the Edu because of the "org.matrix.context" bug you pointed out earlier but it's still clear that this is useful.)